content/docs/guides/parallel-execution.md

```diff
@@ -0,0 +1,396 @@
+
+
+# CLI
+The `Main class` in the `cucumber.api.cli package` is used to execute the feature files. There is no need to create any runner class. The usage options for this class is mentioned [here](https://github.com/cucumber/cucumber-jvm/blob/v4.0.0/core/src/main/resources/cucumber/api/cli/USAGE.txt). The `--threads` option needs to be set to a value **greater than 1** to run in parallel. When the parallel mode is used, the scenarios and rows in a scenario outline will be run in multiple threads.
+ 
+Below is the basic command to start the execution.
+```shell
+java -cp <classpath> cucumber.api.cli.Main -g <steps package> --threads <thread count> <path to feature files>
+```
+
+Follow the below steps to **execute the command from a terminal**.
+
+- Download the necessary cucumber jars from the cucumber.io Maven repository (links provided under version number) or, if available, copy from the local repository into a folder. These need to be provided in the **classpath**. The versions mentioned are compatible with the latest cucumber release 4.3.0.
+
+	1.  `cucumber-core.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-core/4.3.0/cucumber-core-4.3.0.jar)
+	2.  `cucumber-java.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-java/4.3.0/cucumber-java-4.3.0.jar)
+	3.  `cucumber-java8.jar` [`4.3.0`](http://central.maven.org/maven2/io/cucumber/cucumber-java8/4.3.0/cucumber-java8-4.3.0.jar)
+	4.  `cucumber-html.jar` [`0.2.7`](http://central.maven.org/maven2/io/cucumber/cucumber-html/0.2.7/cucumber-html-0.2.7.jar)
+	5.  `gherkin.jar` [`5.1.0`](http://central.maven.org/maven2/io/cucumber/gherkin/5.1.0/gherkin-5.1.0.jar)
+	6.  `cucumber-expressions.jar` [`6.2.2`](http://central.maven.org/maven2/io/cucumber/cucumber-expressions/6.6.2/cucumber-expressions-6.6.2.jar)
+	7.  `datatable.jar` [`1.1.12`](http://central.maven.org/maven2/io/cucumber/datatable/1.1.12/datatable-1.1.12.jar)
+	8.  `datatable-dependencies.jar` [`1.1.12`](http://central.maven.org/maven2/io/cucumber/datatable-dependencies/1.1.12/datatable-dependencies-1.1.12.jar)
+	9.  `tag-expressions.jar` [`1.1.1`](http://central.maven.org/maven2/io/cucumber/tag-expressions/1.1.1/tag-expressions-1.1.1.jar)
+	10. `typetools.jar` [`0.5.0`](http://central.maven.org/maven2/net/jodah/typetools/0.5.0/typetools-0.5.0.jar)
+	
+- Create a folder named `cukecli` and a folder `parallel` inside it, to keep the step definition classes and feature files.
+
+- Create the **two feature files** (`scenarios.feature` and `scenariooutlines.feature`) inside the `parallel` folder as described in the JUnit section.
+
+- Create the **step definition class** in the `parallel` folder as described in the JUnit section.
+
+- Open up a **terminal window** and navigate to the source folder of the project, in our case `cukecli`.
+
+{{% block "java" %}}

```

Comment:
When the content of the language blocks is the same, you can specify multiple languages in the same block, separated by a comma, like this `{{% text "java,kotlin" %}}` (see [Contributing to the documentation](https://cucumber.io/docs/community/contributing-to-documentation/#paragraphs-or-text-that-is-valid-for-multiple-programming-languages))

Reply:
Will change this